demo/android/STT/README.md

```diff
@@ -0,0 +1,19 @@
+# Android Speech-To-Text Demo
+
+This Android demo runs Porcupine with Android STT.
+
+## Usage
+
+Launch the demo on your phone using Android Studio.
```

Comment:
Open `demo/android/STT` with Android Studio. Launch the demo on your connected Android device or simulator.